content/en/real_user_monitoring/roku/data_collected.md

```diff
@@ -0,0 +1,249 @@
+| Resource   | 15 days   | A resource represents network requests to first-party hosts, APIs, and third-party providers in your mobile application. All requests generated during a user session are attached to the view with a unique `resource.id`.                                                                                                                                                                                                                                                           |
+| Error      | 30 days   | An error represents an exception or crash emitted by the mobile application attached to the view it is generated in.                                                                                                                                                                                                                                                                                                                                                                  |
+| Action     | 30 days   | An action represents user activity in your mobile application (such as an application launch, tap, swipe, or back). Each action is attached with a unique `action.id` attached to the view it gets generated in.                                                                                                                                                                                                                                                                      |
+| Long Task  | 15 days   | A long task event is generated for any task in the application that blocks the main thread for more than the specified duration threshold.                                                                                                                                                                                                                                                                                                                                            |
+
+The following diagram illustrates the RUM event hierarchy:
+
+{{< img src="real_user_monitoring/data_collected/event-hierarchy.png" alt="RUM Event hierarchy" style="width:50%;" >}}
+
+## Default attributes
+
+RUM collects common attributes for all events and attributes specific to each event listed below automatically. You can also choose to enrich your user session data by tracking [additional events][2] or by [adding custom attributes][3] to default events specific to your application monitoring and business analytics needs.
```

Comment:
Don't know if this is expected at this point, but the links (not only these, but most of the others) are not showing properly in the preview